src/types/index.ts

```diff
@@ -0,0 +1,41 @@
+import { AWSError, Request } from "aws-sdk";
+import { DescribeStacksOutput } from "aws-sdk/clients/cloudformation";
+
+export type parameterValue = "Environment";
+export type EventName = "UpdateStack" | "CreateStack" | "DeleteStack";
+
+export type parameterKey = {
+  [parameterKey: string]: parameterValue;
+};
+
+export enum CONST {
```

Comment:
Typescript enum convention would be `Const` instead of `CONST`

Reply:
done.